templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,57 @@
-Get Zulip notifications for your Git repositories!
+# Zulip Git integration
+
+Get Zulip notifications for pushes to your Git repositories!
+
+!!! tip ""
+
+    Zulip offers integrations for
+    [various repository hosting services][other-integrations], and also
+    supports this direct Git integration for unsupported services.
+
+!!! warn ""
+
+    This integration is meant to be run on the server-side.
```

Comment:
I assume that this means it needs to be set up on the Zulip server? And therefore needs to be a server administrator who sets up the integration. If so, that should be more clearly stated. And if not, then this should be clearer on what "server-side" means.